applications/nrf5340_audio/README.rst

```diff
@@ -870,19 +870,12 @@ FOTA build files
 
 The generated FOTA build files use the following naming patterns:
 
-* For multi-image DFU, the file name refers to ``dfu_application`` and includes *<Role>*, *<Application version>*, and *<Revision>* in the file name (``dev_<Role>_build_<Application version>_dfu_application_<Revision>.zip``, for example :file:`dev_headset_build_debug_dfu_application_0.0.0+0.zip`).
+* For multi-image DFU, the file name refers to ``dfu_application.zip``.
   This file updates two cores with one single file.
-* For single-image DFU, the bin file for the application core refers in the name to ``app_update`` and includes *<Role>*, *<Application version>*, and *<Revision>* in the file name (``dev_<Role>_build_<Application version>_app_update_<Revision>.bin``, for example :file:`dev_headset_build_debug_app_update_0.0.0+0.bin`).
-  The bin file for the network core refers in the name to ``net_core_app_update`` and includes *<FW_VERSION>* instead of *<Revision>* in the file name (``dev_<Role>_build_<Application version>_net_core_app_update_<FW_VERSION>.bin``, for example :file:`dev_headset_build_debug_net_core_app_update_1.bin`).
+* For single-image DFU, the bin file for the application core refers in the name to ``app_update.bin``.
```

Comment:
Should we even mention single image DFU?

Reply:
> Should we even mention single image DFU?

I think so, we still have internal layout option.

Reply:
Could we then have a disclaimer here as well? Or link to the documentation? We really do not want to use this approach.

Reply:
We have already stated that in caution part.
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-9275/nrf/applications/nrf5340_audio/README.html#configuring-fota-upgrades